datascience/tables.py

```diff
@@ -1610,34 +1613,38 @@ def with_column(self, label, values, *rest):
         """
         # Ensure that if with_column is called instead of with_columns;
         # no error is raised.
-        if rest:
-            return self.with_columns(label, values, *rest)
 
         new_table = self.copy()
-        new_table.append_column(label, values)
+        if formatter == {}:
+            formatter = None
+        elif isinstance(formatter, dict):
+            formatter = formatter["formatter"]
+        new_table.append_column(label, values, formatter)
         return new_table
 
-    def with_columns(self, *labels_and_values):
+    def with_columns(self, *labels_and_values, **formatter):
         """Return a table with additional or replaced columns.
 
 
         Args:
-            ``labels_and_values``: An alternating list of labels and values or
-                a list of label-value pairs. If one of the labels is in
+            ``labels_and_values_and_formats``: An alternating list of labels, values, and (optionally) formatters or
+                a list of label-value pairs and/or label-value-formatter trios. If one of the labels is in
                 existing table, then every value in the corresponding column is
```

Comment:
There is no argument named `labels_and_values_and_formats`.

Reply:
Oops, sorry. Made the correction!